docs/howto/local-checkouts.md

```diff
@@ -0,0 +1,343 @@
+| **clone**                                | Clones lakeFS data from a path into an empty local directory and initializes the directory                                                          | A directory can only track a single lakeFS remote location. i.e., you cannot clone data into an already initialized directory                                                                                             |
+| **list**                                 | Lists directories that are synced with lakeFS | It is recommended to follow any _init_ or _clone_ command with a list command to verify its success |                                                                                                                                                                                                                           | 
+| **status**                               | Shows remote and local changes to the directory and the remote location it tracks                                                                   |                                                                                                                                                                                                                           | 
+| **commit**                               | Commits changes from local directory to the lakeFS branch it tracks                                                                                 | Uncommitted changes to directories connected to lakeFS remote locations will not reflect in lakeFS until after doing lakectl local _commit_.                                                                              | 
+| **pull**                                 | Fetches latest changes from a lakeFS remote location into a connected local directory                                                               |                                                                                                                                                                                                                           |
+| **checkout**                             | Syncs a local directory with the state of a lakeFS ref                                                                                              |                                                                                                                                                                                                                           |
+
+For additional information see the [CLI reference](../reference/cli.md#lakectl-local). 
+
+**Note:** The data size you work with locally should be reasonable for smooth operation on a local machine.
```

Comment:
add a rule of thumb... is 500TB reasonable?

Reply:
A number, even if a ballpark so that less technically capable readers will understand. Could be fixed later though.